cmake/OpenCVFindFrameworks.cmake

```diff
@@ -32,6 +32,16 @@ if(WITH_OPENMP)
   if(OPENMP_FOUND)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+    # NOTES:
+    # 1. OpenMP_CXX_INCLUDE_DIRS is defined since CMake >= 3.16.0.
+    # 2. OpenMP_CXX_LIBRARIES is defined since CMake >= 3.9.0.
+    # 2. For gnu openmp (libgomp, e.g. on Linux), OpenMP_CXX_INCLUDE_DIRS is null
```

Comment:
double `2.`

---

Not sure if we really need such detailed comment here (especially not sure about its accuracy).

> `OpenMP_CXX_LIBRARIES` and `OpenMP_CXX_INCLUDE_DIRS` are available with modern CMake scripts only. We use checks to detect that case.

Reply:
Okay, I will copy-paste all the comments in the first comment of this PR and delete them from the code.

Reply:
Done.